Float.Core/UX/INavigationContext.cs

```diff
@@ -59,7 +60,7 @@ public interface INavigationContext
         /// Dismisses a modally-presented page.
         /// </summary>
         /// <param name="animated">Whether to animate the transition.</param>
-        void DismissPage(bool animated = true);
+        Task DismissPage(bool animated = true);
```

Comment:
If we make one of these awaitable, I'm inclined to make all methods in this interface awaitable. Thoughts?
(Also, the documentation would need to be updated here.)